### ISSUE TITLE
Attachments page title

### ISSUE THREAD

**Issue by Natchoice** [buddypress/bp-attachments#59]
The attachments page title is showing as browse despite having a media gallery tile in the admin

**Comment by imath** [comment]
Hi,
Thanks for your feedback, I'm not sure to understand the issue, searching for 'browse' led me to no output made about it, see: https://github.com/search?q=repo%3Abuddypress%2Fbp-attachments%20browse&type=code

Are you testing with the latest [beta version](https://github.com/buddypress/bp-attachments/releases/tag/1.0.0-beta2)?

Can you add a screenshot about it?

**Comment by Natchoice** [comment]
Yes updated before posting because I thought latest beta might fix. Regardless of whether I use an existing page or a new one for the public attachments page it browse magazine in the title despite title being set to otherwise.

Actual title 

![](https://pasteboard.co/vFGDTnUHLgv6.jpg)

Browser title

![](https://pasteboard.co/okI9Lin6RF8L.jpg)

**Comment by Natchoice** [comment]
Images are showing up

**Comment by imath** [comment]
I'm sorry it looks like no images were inserted into your comment, could you try to drag and drop them again ?

**Comment by Natchoice** [comment]
![3D8C0A15-0EA8-43EF-B3B7-A4B35A4C5C4B](https://user-images.githubusercontent.com/125836530/226489132-5f4e0502-fa60-4263-884f-59a605a36201.jpeg)
![7AE18727-D3E4-4858-84DC-A306301BA52B](https://user-images.githubusercontent.com/125836530/226489135-2fb219d4-2720-4713-8f09-a4d47bd18931.jpeg)
